components/unit/file-explorer/viewer/UnitFileExplorerViewerCsv.vue

```diff
@@ -28,11 +28,24 @@ export default {
         await this.getContentFromFilename(filename)
       },
       immediate: true
+    },
+    loading: {
+      handler(value) {
+        this.$emit('loading', value)
+      },
+      immediate: true
```

Comment:
@andreaskundig 
I got the hourglass to work without the need for a timeout. The trick was to emit the loading event from a watcher with `immediate: true`